modules/assim.batch/R/autoburnin.R

```diff
@@ -46,7 +46,7 @@ getBurnin <- function(jags_out,
   } else {
     index <- utils::tail(which(rowSums(gbr_exceed) > 0), 1) + 1
     stopifnot(length(index) == 1,
-              class(index) %in% c("numeric", "integer"))
+               inherits(("numeric", "integer"),class(index)))
```

Comment:
Invalid R syntax. Should probably be `c("numeric", "integer")`. Also, spacing after comma.

Reply:
Also, `inherits` syntax is `inherits(object, classes)` -- here, it is backwards.